src/components/SearchBar.js

```diff
@@ -0,0 +1,32 @@
+import "./SearchBar.css";
+
+export default function SearchBar({
+  onSubmit,
+  resultsText,
```

Comment:
this here is not the same that you are passing from App js.
```suggestion
  resultsText, filterText
```